docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -1,77 +1,205 @@
 
-1. Create CheckBox using the default constructor:
+To create checkbox using property, follow these steps:
 
-    ```cs
-    CheckBox utilityCheckBox = new CheckBox();
-    ```
+1. Create an instance of a `CheckBox` class using the default constructor:
+   ```cs
+   CheckBox _checkBox = new CheckBox();
+   ```
 
-2. Apply style to the CheckBox:
+2. Set the checkbox properties:
+   ```cs
+   // Path to the directory with images
+   string _URL = Tizen.Applications.Application.Current.DirectoryInfo.Resource + "images/";
 
-    ```cs
-    ButtonStyle utilityStyle = new ButtonStyle
+   _checkBox.Size = new Size(200, 200);
+   _checkBox.ParentOrigin = ParentOrigin.Center;
+   _checkBox.PositionUsesPivotPoint = true;
+   _checkBox.PivotPoint = PivotPoint.Center;
+
+   // Set the icon images for different checkbox states 
+   StringSelector _iconURL = new StringSelector()
+   {
+      Normal           = _URL + "blue.png",
+      Selected         = _URL + "blue_checked.png",
+      Pressed          = _URL + "red.png",
+      Disabled         = _URL + "green.png",
+      DisabledSelected = _URL + "yellow.png"
+   };
+   _checkBox.IconURLSelector = _iconURL;
+   _checkBox.Icon.Size = new Size2D(160,160);
+   _checkBox.BackgroundColor = new Color(0.57f, 0.7f, 1.0f, 0.8f);
+
+   // CheckBox initial state set to be selected
+   _checkBox.IsSelected = true;
+   ```
+
+   To set the absolute path of the images that are used, the `Tizen.Applications.Application.Current.DirectoryInfo.Resource` path is used. For more information, see [Class Application](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.Application.html) and [Class DirectoryInfo](https://samsung.github.io/TizenFX/latest/api/Tizen.Applications.DirectoryInfo.html).
+
+3. Add checkbox to the view:
+   ```cs
+   _rootView.Add(_checkBox);
+   ```
+
+The following output is generated when the checkbox is created using properties:
+
+![CheckBox_prop](./media/CheckBox_prop.gif) 
+
+A checkbox selection can be disabled by adding the following code:
+   ```cs
+   // CheckBox can not be selected 
+   _checkBox.IsSelectable = false;
+   ```
+
+To disable the checkbox completely use the following code:
+   ```cs
+   // CheckBox is disabled
+   _checkBox.IsEnabled = false;
+   ```
+This sets the checkbox state to `Disabled` or `DisabledSelected` depending on the `IsSelected` value. The preceding options change the checkbox appearance as follows:
+
+| ![CheckBox_dis](./media/CheckBox_dis.gif)                   | ![CheckBox_dis](./media/CheckBox_yellow.gif)             | ![CheckBox_dis](./media/CheckBox_green.gif)               |
+|:-----------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------------------:|
+| ```IsSelectable``` set to `false`<br>`IsSelected` set to `true` | `IsEnabled` set to `false`<br>`IsSelected` set to `true` | `IsEnabled` set to `false`<br>`IsSelected` set to `false` |
```

Comment:
@annie-samsung I think you missed this.

```suggestion
| `IsSelectable` set to `false`<br>`IsSelected` set to `true` | `IsEnabled` set to `false`<br>`IsSelected` set to `true` | `IsEnabled` set to `false`<br>`IsSelected` set to `false` |
```

Reply:
@mijin85cho, I am still reviewing, looks like I accidently submitted single review or so. Thanks for pointing it out.

Reply:
@annie-samsung I see. Thanks for the reply.